RATIONALE.md

```diff
@@ -0,0 +1,23 @@
+# zipkin-storage-kafka rationale
+
+## Use-cases
+
+### Replacement for batch-oriented Zipkin dependencies
+
+A limitation of [zipkin-dependencies](https://github.com/openzipkin/zipkin-dependencies) module, is that it requires to be scheduled with a defined frequency. This batch-oriented execution causes out-of-date values until processing runs again.
+
+Kafka-based storage enables aggregating dependencies as spans are received, allowing a (near-)real-time calculation of dependency metrics.
```

Comment:
this whole part could be in a readme instead. the part from this line up is good for rationale. we dont need how to in rationed in other words